core/src/main/scala/com/databricks/labs/remorph/generators/sql/LogicalPlanGenerator.scala

```diff
@@ -34,12 +34,71 @@ class LogicalPlanGenerator(val expr: ExpressionGenerator, val explicitDistinct:
     case ir.DeleteFromTable(target, None, where, None, None) => delete(ctx, target, where)
     case c: ir.CreateTableCommand => createTable(ctx, c)
     case t: ir.TableSample => tableSample(ctx, t)
+    case a: ir.AlterTableCommand => alterTable(ctx, a)
     case ir.NoopNode => ""
     case ir.UnresolvedCommand(inputText) => s"--${inputText}"
     case null => "" // don't fail transpilation if the plan is null
     case x => throw unknown(x)
   }
 
+  private def alterTable(ctx: GeneratorContext, a: ir.AlterTableCommand): String = {
+    val operation = buildTableAlteration(ctx, a.alterations)
+    s"ALTER TABLE ${a.tableName} $operation"
+  }
+
+  private def buildTableAlteration(ctx: GeneratorContext, alterations: Seq[ir.TableAlteration]): String = {
+    alterations.headOption match {
+      case Some(a: ir.AddColumn) => buildAddColumn(ctx, alterations.collect { case c: ir.AddColumn => c })
```

Comment:
this code is incorrect and is going to break when there's more than one alteration.